README.md

```diff
@@ -1,48 +1,53 @@
+## Our images
 
-As an example scenario, imagine that you made a python script and you want your friend to be able to run it. This python script relies on a couple different packages, that you downloaded on to your computer through various means. Now you send it to your friend and they try and run it, but then tell you it won't run, because they don't have any of the packages necessary. You can solve this in two ways. The first is to tell them how to install everything, send them links to where you got the packages, etc. Oh also, they're running Windows and you're on Ubuntu. You then realize that one of the packages only exists for Windows, and you start searching for a replacement.
-Now the second option: Docker. What you can do is create a Dockerfile that inherits from Ubuntu, and then you can write out a series of instructions containing all the steps you followed to create your environment. For example, you can put all the `apt-get install`s in this Dockerfile. Then you can build this Dockerfile into an image, and put the image online. Now, all your friend on Windows has to do is download your image, create a container, and use the container to run your code. This means that your friend does not need to do any installation, and also runs the code in the exact same environment as you, despite being on a different computer.
+- `base` - built from NVIDIA's images to fix their incorrect architecture tagging
+- [`core`](core) - built on `base`, never run, contains common components for `onboard` and `landside`
+- [`onboard`](onboard) - built on `core`, run on the robot itself
+- [`landside`](landside) - built on `core`, run on the landside computer during a pool test
 
-Robot Operating System (ROS) is quite hefty and does not run easily on all machines. Our solution to this was to create a docker image with ROS inside it, so that for users to run ROS, they just need to install Docker and download our image.
 
-## Building
-Building a docker container is the process of taking a Dockerfile, running all the commands inside it, and creating an image. Because we need to build our Dockerfile for multiple platforms (x86 for most computers and ARM for the Jetson), we use the `buildx` command, which allows us to build for multiple platforms at the same time. For more information on buildx, go [here](https://docs.docker.com/buildx/working-with-buildx/).
+## Building manually
 
-Before trying to use buildx, make sure you have experimental features enabled. Go to Docker Preferences -> Command Line and select 'Enable experimental features'
+:warning: Our automatic build means that **you should almost never need to do a manual build.** But if you do, read on.
+
+Since we need to build our Dockerfile for multiple platforms (x86 for most computers and ARM for the NVIDIA Jetson TX2), we use the `buildx` command, which allows us to build for multiple platforms at the same time. [More information on buildx](https://docs.docker.com/buildx/working-with-buildx/).
```

Comment:
nit:
```suggestion
Since we need to build our Dockerfile for multiple platforms (x86 for most computers and ARM for the NVIDIA Jetson TX2), we use the `buildx` command, which allows us to build for multiple platforms simultaneously. [More information on buildx](https://docs.docker.com/buildx/working-with-buildx/).
```

Reply:
Done.